### DISCUSSION TITLE
Hotkey to set cursor before bloc of code

### DISCUSSION THREAD

**Discussion by wondertalik** [ueberdosis/tiptap#2184] [General Questions]
Hey there. How can i put new line before bloc of code? If my cursor in this bloc i can press `cmd + enter` to set cursor after.

![image](https://user-images.githubusercontent.com/1611064/142827307-150c1581-0e15-4956-9316-d96874d78556.png)

**Comment by cadars** [comment]
Maybe it's possible to extend the codeBlock extension and specify `allowGapCursor: true` ?
(just a guess, I haven't tried)
